x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_about_rule/schema.tsx

```diff
@@ -100,6 +109,28 @@ export const schema: FormSchema = {
       }
     ),
     labelAppend: <EuiText size="xs">{RuleI18n.OPTIONAL_FIELD}</EuiText>,
+    validations: [
+      {
+        validator: (
+          ...args: Parameters<ValidationFunc>
+        ): ReturnType<ValidationFunc<{}, ERROR_CODE>> | undefined => {
+          const [{ value, path }] = args;
+          let hasError = false;
+          (value as string[]).forEach(url => {
```

Comment:
nit: Types are a little loose here -- potential improvement to the `es_ui_shared` `ValidationFunc` interface.